### PR TITLE
Fix admin overlay message too long

### PR DESCRIPTION
**Pull request by NoooneyDude**
CL: [Fix] Fix admins getting kicked from too many admin info updates by making the network messages more efficient. Might still occur in the future, which will require a refactor, but let's see if that's required.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Messages/Server/AdminTools/AdminInfoUpdateMessage.cs [hunk 1]
```diff
@@ -1,86 +1,77 @@
...
+using AdminTools;
 
 namespace Messages.Server.AdminTools
 {
 	public class AdminInfoUpdateMessage : ServerMessage<AdminInfoUpdateMessage.NetMessage>
 	{
 		public struct NetMessage : NetworkMessage
 		{
-			public string JsonData;
+			public AdminInfosEntry[] Entries;
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Messages/Server/AdminTools/AdminInfoUpdateMessage.cs:14]: does Weaver support (Mirror) arrays 
I thought it didn't?

Comment (NoooneyDude) [reply UnityProject/Assets/Scripts/Messages/Server/AdminTools/AdminInfoUpdateMessage.cs:14]: It does!

https://mirror-networking.gitbook.io/docs/guides/data-types

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Messages/Server/AdminTools/AdminInfoUpdateMessage.cs:14]: oh ok, as long as it's a struct with basic types cool cool

**Review by Bod9001 — APPROVED** [review]

File: UnityProject/Assets/Scripts/Messages/Server/AdminTools/AdminInfoUpdateMessage.cs (hunk 1 above)
Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Messages/Server/AdminTools/AdminInfoUpdateMessage.cs:14]: Arrays of structs of accepted types are fine

**Review by ThatDan123 — APPROVED** [review]

**Pull request merged**